plugins/woocommerce/src/Internal/ComingSoon/ComingSoonRequestHandler.php

```diff
@@ -83,4 +92,10 @@ private function render_404() {
 		}
 		die();
 	}
+
+	private function maybe_set_cookie() {
```

Comment:
It seems that this method is unused. Should we remove it?

Reply:
Yup good idea, it was an idea I was exploring but we don't need it.  FIxed in 048dd1a